compiler-core/src/build/package_compiler.rs

```diff
@@ -29,13 +29,26 @@ use camino::{Utf8Path, Utf8PathBuf};
 
 use super::{ErlangAppCodegenConfiguration, TargetCodegenConfiguration, Telemetry};
 
+#[derive(Copy, Clone, Debug)]
+/// A flag that can be used to tell the package compiler to drop all the modules
+/// before performing code generation.
```

Comment:
Can we change this from talking about dropping of modules and codegen to explain what the intent is? I find these docs a bit confusing atm, especially since we really want to not load them at all rather than drop them.